app/assets/javascripts/js-processor.js

```diff
@@ -26,6 +26,10 @@ import RawHandlebars from "discourse-common/addon/lib/raw-handlebars";
 import { WidgetHbsCompiler } from "discourse-widget-hbs/lib/widget-hbs-compiler";
 import EmberThisFallback from "ember-this-fallback";
 
+import getRandomValues from "polyfill-crypto.getrandomvalues";
+globalThis.crypto = { getRandomValues };
```

Comment:
Maybe add a comment to explain why this is required? It's not obvious at first glance